doc/nrf/protocols/thread/prebuilt_libs.rst

```diff
@@ -233,18 +234,18 @@ Use this functionality for :ref:`certification <ug_thread_cert>` of your configu
 
 You can install the libraries either with or without debug symbols.
 Installing the libraries with debug symbols can be useful when debugging, but will take a significant amount of storage memory.
-You can remove the symbols when updating with the :kconfig:option:`CONFIG_OPENTHREAD_BUILD_OUTPUT_STRIPPED` Kconfig option enabled.
-The option is disabled by default.
 
 .. note::
-    When you select :kconfig:option:`CONFIG_OPENTHREAD_USER_CUSTOM_LIBRARY`, the location of the destination directory for the libraries depends on the chosen :ref:`nrf_security backend <nrf_security_readme>`, either :kconfig:option:`CONFIG_CC3XX_BACKEND` or :kconfig:option:`CONFIG_OBERON_BACKEND`.
+   When you select :kconfig:option:`CONFIG_OPENTHREAD_USER_CUSTOM_LIBRARY`, the location of the directory for the libraries depends on the chosen :ref:`nrf_security backend <nrf_security_readme>`, either :kconfig:option:`CONFIG_CC3XX_BACKEND` or :kconfig:option:`CONFIG_OBERON_BACKEND`.
+   For example, if you choose :kconfig:option:`CONFIG_OBERON_BACKEND`, the directory is :file:ncs/nrfxlib/openthread/lib/cortex-m33/hard-float/v1.3/master/oberon`
```

Comment:
I think it would be better to be more clear that this is an example path, eg. "For example, if you choose :kconfig:option:`CONFIG_OBERON_BACKEND`, the directory might look like: :file:ncs/nrfxlib/openthread/lib/cortex-m33/hard-float/v1.3/master/oberon`.

Reply:
Discussed offline, and applied a fix.